tests/edgeless/shape.spec.ts

```diff
@@ -666,19 +666,24 @@ test.describe('shape hit test', () => {
     await assertEdgelessCanvasText(page, 'hello');
   });
 
-  test('using text tool to add text in shape hollow area', async ({ page }) => {
-    await addTransparentRect(page, rect.start, rect.end);
-    await page.mouse.click(rect.start.x - 20, rect.start.y - 20);
-    await assertEdgelessNonSelectedRect(page);
-
-    await assertEdgelessTool(page, 'default');
-    await setEdgelessTool(page, 'text');
-    await page.mouse.click(rect.start.x + 20, rect.start.y + 20);
-    await waitNextFrame(page);
-
-    await type(page, 'hello');
-    await assertEdgelessCanvasText(page, 'hello');
-  });
+  // FIXME(@flrande): This is broken by recent changes
+  // In Playwright, we can't add text in shape hollow area
+  test.fixme(
```

Comment:
@Flrande This is broken on master. It works in playground but in playwright, line 680 won't create edgeless text element on shape for some reasons.